src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/ReportedLayer/ReportedLayer.tsx

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+import { useCallback, useContext } from 'react'
+import L from 'leaflet'
+import '../style.css'
+
+import type { FeatureCollection } from 'geojson'
+import type { FC } from 'react'
+import type {
+  DataLayerProps,
+  Feature,
+} from 'signals/incident/components/form/MapSelectors/Asset/types'
+import { Marker } from '@amsterdam/arm-core'
+import { getIconUrl } from 'signals/incident/components/form/MapSelectors/utils'
+import WfsDataContext from '../context'
+
+const REPORTED_CLASS_MODIFIER = 'marker-reported'
+
+const ReportedLayer: FC<DataLayerProps> = ({ featureTypes }) => {
+  const data = useContext<FeatureCollection>(WfsDataContext)
+
+  const getFeatureType = useCallback(
+    (feat: any) => {
+      const feature = feat as Feature
+      if (feature.properties.meldingstatus === 1) {
+        return featureTypes.find(({ typeValue }) => typeValue === 'reported')
+      }
+    },
+    [featureTypes]
+  )
+
+  const getMarker = useCallback(
+    (feat: any) => {
+      const feature = feat as Feature
+      const [lng, lat] = feature.geometry.coordinates
+      const latLng = { lat, lng }
+      const featureType = getFeatureType(feature)
+      if (!featureType) return
+      const iconSvg = featureType && featureType.icon.iconSvg
+
+      const iconSize = [20, 20] as [number, number]
```

Comment:
Is the type casting required? Can't TS infer the types?

Reply:
TS can not

Reply:
`L.icon` expects a `PointTuple` as a value for `iconSize`. Maybe use that?

Reply:
I solved it like this:

`
      const icon = L.icon({
        iconSize: [20, 20],
        iconUrl: getIconUrl(iconSvg),
        className: REPORTED_CLASS_MODIFIER,
      })
`